docs/features/content-extensions/contentManager.md

```diff
@@ -0,0 +1,40 @@
+---
+id: contentManager
+title: ContentManager
+sidebar_label: ContentManager
+description: Extensions for the MonoGame ContentManager class.
+---
+
+:::tip[Up to date]
+This page is **up to date** for MonoGame.Extended `@mgeversion@`.  If you find outdated information, [please open an issue](https://github.com/craftworkgames/craftworkgames.github.io/issues).
+:::
+
+# ContentManager extensions
+
+## ContentManager.OpenStream
+
+`System.IO.Stream ContentManager.OpenStream(string filename)`
+
+**OpenStream** allows easy access to `TitleContainer.OpenStream` so you can use the `Game.Content` object to load compiled resources *and* included resources.
+
+Example below loads a text file "song-lyrics" directly.  This file is assumed to have it's properties set to "Copy to Output Directory".
+```csharp
+// song-lyrics.txt is in the Content Directory
+var stream = Content.OpenStream("song-lyrics.txt");
+var reader = new StreamReader(stream);
+string songLyrics = reader.ReadToEnd();
+
+reader.Close();
+stream.Close();
```

Comment:
Might be better practice to use `using` statements here instead of manually closing out streams after.

```cs
using var stream = Content.OpenStream("song-lyrics.txt");
using var reader = new StreamReader(stream);
string songLyrics = reader.ReadToEnd();
```

Or the traditional way with brackets and indentations

```cs
string songLyrics = string.Empty;

using(var stream = Content.OpenStream("song-lyrics.txt"))
{
    using(var reader = new StreamReader(stream))
    {
        songLyrics = reader.ReadToEnd();
    }
}
```

Otherwise, the only reason to manually close the streams out is if there is error handling implemented

```cs
string songLyrics = string.Empty;
Stream stream;
StreamReader reader;

try
{
    stream = Content.OpenStream("song-lyrics.txt");
    reader = new StreamReader(stream);
    
    songLyrics = reader.ReadToEnd();
}
catch(Exception ex)
{
    // do something with exception
}
finally
{
    stream.Close();
    reader.Close();
}
```

Reply:
Ok, I had written it with brackets and indentations originally, but then I figured I should keep it closer to the original example.

I'll update it to use the `using declaration` format using var blah = stuff();  I had actually never seen this format before and had to lookup how the scope works.

Do you have a preference?  It felt odd to me at first that brackets were missing (Old school)

Reply:
I'm old school too, and prefer the brackets when possible.  I think for the example snippet, with brackets would be better.

For reference though, the new style without brackets performs the `IDisposable` call on the using object when the method scope it's used in goes out of scope.